Missing include required by snprintf and --disable-openssl ./configure switch.

diff -ur libtorrent-0.12.6.orig/src/protocol/handshake.cc libtorrent-0.12.6/src/protocol/handshake.cc
--- libtorrent-0.12.6.orig/src/protocol/handshake.cc	2009-11-12 10:03:51.000000000 +0200
+++ libtorrent-0.12.6/src/protocol/handshake.cc	2009-12-22 22:05:27.000000000 +0200
@@ -53,6 +53,8 @@
 #include "handshake.h"
 #include "handshake_manager.h"
 
+#include <stdio.h>
+
 namespace torrent {
 
 const char* Handshake::m_protocol = "BitTorrent protocol";
